Makefile

```diff
@@ -176,6 +176,13 @@ out/minikube_$(DEB_VERSION).deb: out/minikube-linux-amd64
 	dpkg-deb --build out/minikube_$(DEB_VERSION)
 	rm -rf out/minikube_$(DEB_VERSION)
 
+out/minikube_$(DEB_VERSION).tar.gz: out/localkube out/minikube-linux-amd64 out/minikube-darwin-amd64 out/minikube-windows-amd64.exe
```

Comment:
I dont think we should version it in the name
I'm also more for having each platform have its own tarball `minikube-linux-amd64.tar.gz`, etc.